app/command_test.go

```diff
@@ -75,6 +75,29 @@ func TestCreateCommandPost(t *testing.T) {
 	}
 }
 
+func TestExecuteCommand(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+
+	args := &model.CommandArgs{
+		Command: "/code happy path",
+	}
+
+	resp, err := th.App.ExecuteCommand(args)
+
+	assert.Nil(t, err)
+	assert.Equal(t, resp.Text, "happy path")
+
+	args = &model.CommandArgs{
+		Command: "/code\nnewline path",
```

Comment:
I'd love to see expanded unit tests cases that test a variety of inputs, ideally using table tests to setup and define. It doesn't hurt to exhaust the potential whitespace characters as tests, as well as look for other interesting edge cases.

Reply:
@lieut-data  its currently working when testing manually with web app.  Still working on Unit tests to get it to pass this basic case.  Still working on trying to set up delve with VS Code so i can debug.
    
Is there any good examples in the code base for using table tests?

Reply:
For sure! Here's a good example: https://github.com/mattermost/mattermost-server/blob/ebfd332161cc8a8d62160964a5e96b82383becef/app/authentication_test.go#L16-L52